src/main.tsx

```diff
@@ -1,9 +1,9 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.tsx'
+import { StrictMode } from "react";
```

Comment:
I suggest to keep the whole app without semicolons and with single quotes instead of double quotes. It just looks cleaner in my opinion. For this there are ESLint rules that we can add in the config. If you then apply the "lint on save" feature in your IDE it would automatically pick up the new rules.